configs/config.json.example

```diff
@@ -153,6 +153,7 @@
         "type": "EvolvePokemon",
         "config": {
           "enabled": false,
+          "interval": 60,

```

Comment:
@alexyaoyang I've added the documentation line. I have been trying this at home for 2 days now and I feel much more comfortable with 120 as the default value because we don't want this message to be spammy 😄